plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/util/DataSkippingStatsColumnsUtils.java

```diff
@@ -0,0 +1,226 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.deltalake.util;
+
+import com.google.common.base.Joiner;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableSet;
+import com.google.common.escape.Escaper;
+import com.google.common.escape.Escapers;
+import io.trino.spi.TrinoException;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.Set;
+
+import static io.trino.plugin.deltalake.DeltaLakeErrorCode.DELTA_LAKE_INVALID_SCHEMA;
+import static io.trino.plugin.deltalake.transactionlog.DeltaLakeSchemaSupport.DATA_SKIP_STATS_COLUMN_CONFIGURATION_KEY;
+import static java.lang.String.format;
+
+public final class DataSkippingStatsColumnsUtils
+{
+    private static final String SPECIAL_CHARS = "!@#$%^&*()+-={}|[]:\\\";'<>,.?/";
```

Comment:
Where is the source of these characters?

Reply:
It come from the delta [pr](https://github.com/delta-io/delta/pull/1763), and the comment in [here](https://github.com/chenjian2664/delta/blob/56cb730f0bd0639018d09c16546ea88c5211c2f3/spark/src/main/scala/org/apache/spark/sql/delta/DeltaConfig.scala#L596)